tests/unit/module_utils/test_ec2.py

```diff
@@ -232,3 +270,34 @@ def test_compare_boolean_policy_bool_and_string_are_equal(self):
     def test_compare_numeric_policy_number_and_string_are_equal(self):
         """ Testing two policies one using a quoted number, the other an int """
         self.assertFalse(compare_policies(self.numeric_policy_string, self.numeric_policy_number))
+
+    def test_compare_version_policies_defaults_old(self):
+        """ Testing that a policy without Version is considered identical to one
+        with the 'old' Version (by default)
+        """
+        self.assertFalse(compare_policies(self.version_policy_old, self.version_policy_missing))
+        self.assertTrue(compare_policies(self.version_policy_new, self.version_policy_missing))
+
+    def test_compare_version_policies_default_disabled(self):
+        """ Testing that a policy without Version not considered identical when default_version=None
+        """
+        self.assertFalse(compare_policies(self.version_policy_missing, self.version_policy_missing, default_version=None))
+        self.assertTrue(compare_policies(self.version_policy_old, self.version_policy_missing, default_version=None))
+        self.assertTrue(compare_policies(self.version_policy_new, self.version_policy_missing, default_version=None))
+
+    def test_compare_version_policies_default_set(self):
+        """ Testing that a policy without Version is only considered identical
+        when default_version=
```

Comment:
```suggestion
        when default_version="2008-10-17"
```
I think this is what's meant to be here?